2024/03/21 11:45:23 [error] 176#176: *980 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET / HTTP/2.0", upstream: "http://192.168.1.10:3000/", host: "grafana.xenlor.com"
2024/03/21 11:45:23 [error] 176#176: *980 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET /favicon.ico HTTP/2.0", upstream: "http://192.168.1.10:3000/favicon.ico", host: "grafana.xenlor.com", referrer: "https://grafana.xenlor.com/"
2024/03/21 11:45:27 [error] 176#176: *980 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET / HTTP/2.0", upstream: "http://192.168.1.10:3000/", host: "grafana.xenlor.com"
2024/03/21 11:45:45 [error] 176#176: *980 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET / HTTP/2.0", upstream: "http://192.168.1.10:3000/", host: "grafana.xenlor.com"
2024/03/21 11:45:57 [error] 176#176: *980 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET / HTTP/2.0", upstream: "http://192.168.1.10:3000/", host: "grafana.xenlor.com"
2024/03/21 11:45:58 [error] 176#176: *980 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET / HTTP/2.0", upstream: "http://192.168.1.10:3000/", host: "grafana.xenlor.com"
2024/03/21 11:46:24 [error] 176#176: *980 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET / HTTP/2.0", upstream: "http://192.168.1.10:3000/", host: "grafana.xenlor.com"
2024/03/21 11:47:13 [error] 176#176: *980 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET / HTTP/2.0", upstream: "http://192.168.1.10:3000/", host: "grafana.xenlor.com"
2024/03/21 11:49:17 [error] 176#176: *1343 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET / HTTP/2.0", upstream: "http://192.168.1.10:3000/", host: "grafana.xenlor.com"
2024/03/21 11:49:19 [error] 176#176: *1343 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET / HTTP/2.0", upstream: "http://192.168.1.10:3000/", host: "grafana.xenlor.com"
2024/03/21 11:49:20 [error] 176#176: *1343 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET / HTTP/2.0", upstream: "http://192.168.1.10:3000/", host: "grafana.xenlor.com"
2024/03/21 11:49:22 [error] 176#176: *1343 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET / HTTP/2.0", upstream: "http://192.168.1.10:3000/", host: "grafana.xenlor.com"
2024/03/21 11:49:22 [error] 176#176: *1343 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET / HTTP/2.0", upstream: "http://192.168.1.10:3000/", host: "grafana.xenlor.com"
2024/03/21 11:50:46 [error] 176#176: *1343 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET / HTTP/2.0", upstream: "http://192.168.1.10:3000/", host: "grafana.xenlor.com"
2024/03/21 11:50:46 [error] 176#176: *1343 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET / HTTP/2.0", upstream: "http://192.168.1.10:3000/", host: "grafana.xenlor.com"
2024/03/21 11:51:25 [error] 176#176: *1343 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET / HTTP/2.0", upstream: "http://192.168.1.10:3000/", host: "grafana.xenlor.com"
2024/03/21 11:51:51 [error] 211#211: *1416 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 81.45.67.146, server: grafana.xenlor.com, request: "GET / HTTP/2.0", upstream: "http://192.168.1.10:3000/", host: "grafana.xenlor.com"
2024/03/21 11:51:52 [error] 211#211: *1416 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 81.45.67.146, server: grafana.xenlor.com, request: "GET / HTTP/2.0", upstream: "http://192.168.1.10:3000/", host: "grafana.xenlor.com"
2024/03/21 11:56:19 [error] 224#224: *2155 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET /api/live/ws HTTP/1.1", upstream: "http://192.168.1.10:3000/api/live/ws", host: "grafana.xenlor.com"
2024/03/21 11:56:33 [error] 224#224: *2196 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET /api/live/ws HTTP/1.1", upstream: "http://192.168.1.10:3000/api/live/ws", host: "grafana.xenlor.com"
2024/03/21 11:56:33 [error] 225#225: *2184 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "POST /api/ds/query?ds_type=prometheus&requestId=Q290 HTTP/2.0", upstream: "http://192.168.1.10:3000/api/ds/query?ds_type=prometheus&requestId=Q290", host: "grafana.xenlor.com", referrer: "https://grafana.xenlor.com/?orgId=1&refresh=5s"
2024/03/21 11:56:33 [error] 225#225: *2184 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "POST /api/ds/query?ds_type=prometheus&requestId=Q292 HTTP/2.0", upstream: "http://192.168.1.10:3000/api/ds/query?ds_type=prometheus&requestId=Q292", host: "grafana.xenlor.com", referrer: "https://grafana.xenlor.com/?orgId=1&refresh=5s"
2024/03/21 11:56:33 [error] 225#225: *2184 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "POST /api/ds/query?ds_type=prometheus&requestId=Q293 HTTP/2.0", upstream: "http://192.168.1.10:3000/api/ds/query?ds_type=prometheus&requestId=Q293", host: "grafana.xenlor.com", referrer: "https://grafana.xenlor.com/?orgId=1&refresh=5s"
2024/03/21 11:56:33 [error] 225#225: *2184 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "POST /api/ds/query?ds_type=prometheus&requestId=Q291 HTTP/2.0", upstream: "http://192.168.1.10:3000/api/ds/query?ds_type=prometheus&requestId=Q291", host: "grafana.xenlor.com", referrer: "https://grafana.xenlor.com/?orgId=1&refresh=5s"
2024/03/21 11:56:33 [error] 225#225: *2184 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET /api/annotations?from=1711011278463&to=1711022078463&limit=100&matchAny=false&dashboardUID=VKtkKcAVz HTTP/2.0", upstream: "http://192.168.1.10:3000/api/annotations?from=1711011278463&to=1711022078463&limit=100&matchAny=false&dashboardUID=VKtkKcAVz", host: "grafana.xenlor.com", referrer: "https://grafana.xenlor.com/?orgId=1&refresh=5s"
2024/03/21 11:56:33 [error] 225#225: *2184 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "POST /api/ds/query?ds_type=prometheus&requestId=Q294 HTTP/2.0", upstream: "http://192.168.1.10:3000/api/ds/query?ds_type=prometheus&requestId=Q294", host: "grafana.xenlor.com", referrer: "https://grafana.xenlor.com/?orgId=1&refresh=5s"
2024/03/21 11:56:33 [error] 225#225: *2184 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "POST /api/ds/query?ds_type=prometheus&requestId=Q295 HTTP/2.0", upstream: "http://192.168.1.10:3000/api/ds/query?ds_type=prometheus&requestId=Q295", host: "grafana.xenlor.com", referrer: "https://grafana.xenlor.com/?orgId=1&refresh=5s"
2024/03/21 11:56:33 [error] 225#225: *2184 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "POST /api/ds/query?ds_type=prometheus&requestId=Q296 HTTP/2.0", upstream: "http://192.168.1.10:3000/api/ds/query?ds_type=prometheus&requestId=Q296", host: "grafana.xenlor.com", referrer: "https://grafana.xenlor.com/?orgId=1&refresh=5s"
2024/03/21 11:56:33 [error] 225#225: *2184 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "POST /api/ds/query?ds_type=prometheus&requestId=Q297 HTTP/2.0", upstream: "http://192.168.1.10:3000/api/ds/query?ds_type=prometheus&requestId=Q297", host: "grafana.xenlor.com", referrer: "https://grafana.xenlor.com/?orgId=1&refresh=5s"
2024/03/21 11:56:33 [error] 225#225: *2184 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "POST /api/ds/query?ds_type=prometheus&requestId=Q298 HTTP/2.0", upstream: "http://192.168.1.10:3000/api/ds/query?ds_type=prometheus&requestId=Q298", host: "grafana.xenlor.com", referrer: "https://grafana.xenlor.com/?orgId=1&refresh=5s"
2024/03/21 11:56:33 [error] 225#225: *2184 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "POST /api/ds/query?ds_type=prometheus&requestId=Q299 HTTP/2.0", upstream: "http://192.168.1.10:3000/api/ds/query?ds_type=prometheus&requestId=Q299", host: "grafana.xenlor.com", referrer: "https://grafana.xenlor.com/?orgId=1&refresh=5s"
2024/03/21 11:56:33 [error] 225#225: *2184 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "POST /api/ds/query?ds_type=prometheus&requestId=Q300 HTTP/2.0", upstream: "http://192.168.1.10:3000/api/ds/query?ds_type=prometheus&requestId=Q300", host: "grafana.xenlor.com", referrer: "https://grafana.xenlor.com/?orgId=1&refresh=5s"
2024/03/21 11:56:33 [error] 225#225: *2184 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "POST /api/ds/query?ds_type=prometheus&requestId=Q302 HTTP/2.0", upstream: "http://192.168.1.10:3000/api/ds/query?ds_type=prometheus&requestId=Q302", host: "grafana.xenlor.com", referrer: "https://grafana.xenlor.com/?orgId=1&refresh=5s"
2024/03/21 11:56:33 [error] 225#225: *2184 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "POST /api/ds/query?ds_type=prometheus&requestId=Q301 HTTP/2.0", upstream: "http://192.168.1.10:3000/api/ds/query?ds_type=prometheus&requestId=Q301", host: "grafana.xenlor.com", referrer: "https://grafana.xenlor.com/?orgId=1&refresh=5s"
2024/03/21 11:56:33 [error] 225#225: *2184 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "POST /api/ds/query?ds_type=prometheus&requestId=Q303 HTTP/2.0", upstream: "http://192.168.1.10:3000/api/ds/query?ds_type=prometheus&requestId=Q303", host: "grafana.xenlor.com", referrer: "https://grafana.xenlor.com/?orgId=1&refresh=5s"
2024/03/21 11:56:33 [error] 225#225: *2184 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "POST /api/ds/query?ds_type=prometheus&requestId=Q304 HTTP/2.0", upstream: "http://192.168.1.10:3000/api/ds/query?ds_type=prometheus&requestId=Q304", host: "grafana.xenlor.com", referrer: "https://grafana.xenlor.com/?orgId=1&refresh=5s"
2024/03/21 11:56:33 [error] 225#225: *2184 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "POST /api/ds/query?ds_type=prometheus&requestId=Q305 HTTP/2.0", upstream: "http://192.168.1.10:3000/api/ds/query?ds_type=prometheus&requestId=Q305", host: "grafana.xenlor.com", referrer: "https://grafana.xenlor.com/?orgId=1&refresh=5s"
2024/03/21 11:56:33 [error] 225#225: *2184 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "POST /api/ds/query?ds_type=prometheus&requestId=Q306 HTTP/2.0", upstream: "http://192.168.1.10:3000/api/ds/query?ds_type=prometheus&requestId=Q306", host: "grafana.xenlor.com", referrer: "https://grafana.xenlor.com/?orgId=1&refresh=5s"
2024/03/21 11:56:33 [error] 225#225: *2184 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "POST /api/ds/query?ds_type=prometheus&requestId=Q307 HTTP/2.0", upstream: "http://192.168.1.10:3000/api/ds/query?ds_type=prometheus&requestId=Q307", host: "grafana.xenlor.com", referrer: "https://grafana.xenlor.com/?orgId=1&refresh=5s"
2024/03/21 11:56:33 [error] 225#225: *2184 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "POST /api/ds/query?ds_type=prometheus&requestId=Q308 HTTP/2.0", upstream: "http://192.168.1.10:3000/api/ds/query?ds_type=prometheus&requestId=Q308", host: "grafana.xenlor.com", referrer: "https://grafana.xenlor.com/?orgId=1&refresh=5s"
2024/03/21 11:56:34 [error] 225#225: *2184 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET /?orgId=1&refresh=5s HTTP/2.0", upstream: "http://192.168.1.10:3000/?orgId=1&refresh=5s", host: "grafana.xenlor.com"
2024/03/21 11:56:38 [error] 225#225: *2184 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET /?orgId=1&refresh=5s HTTP/2.0", upstream: "http://192.168.1.10:3000/?orgId=1&refresh=5s", host: "grafana.xenlor.com"
2024/03/21 11:56:39 [error] 225#225: *2184 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET /?orgId=1&refresh=5s HTTP/2.0", upstream: "http://192.168.1.10:3000/?orgId=1&refresh=5s", host: "grafana.xenlor.com"
2024/03/21 11:56:39 [error] 225#225: *2184 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET /?orgId=1&refresh=5s HTTP/2.0", upstream: "http://192.168.1.10:3000/?orgId=1&refresh=5s", host: "grafana.xenlor.com"
2024/03/21 11:56:40 [error] 225#225: *2184 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET /?orgId=1&refresh=5s HTTP/2.0", upstream: "http://192.168.1.10:3000/?orgId=1&refresh=5s", host: "grafana.xenlor.com"
2024/03/21 11:56:41 [error] 225#225: *2184 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET /?orgId=1&refresh=5s HTTP/2.0", upstream: "http://192.168.1.10:3000/?orgId=1&refresh=5s", host: "grafana.xenlor.com"
2024/03/21 12:01:19 [error] 224#224: *2477 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET /?orgId=1&refresh=5s HTTP/2.0", upstream: "http://192.168.1.10:3000/?orgId=1&refresh=5s", host: "grafana.xenlor.com"
2024/03/21 12:01:19 [error] 224#224: *2477 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET /?orgId=1&refresh=5s HTTP/2.0", upstream: "http://192.168.1.10:3000/?orgId=1&refresh=5s", host: "grafana.xenlor.com"
2024/03/21 12:01:21 [error] 224#224: *2477 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET /?orgId=1&refresh=5s HTTP/2.0", upstream: "http://192.168.1.10:3000/?orgId=1&refresh=5s", host: "grafana.xenlor.com"
2024/03/21 12:04:35 [error] 224#224: *2553 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET /?orgId=1&refresh=5s HTTP/2.0", upstream: "http://192.168.1.10:3000/?orgId=1&refresh=5s", host: "grafana.xenlor.com"
2024/03/21 12:05:49 [error] 224#224: *2553 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET / HTTP/2.0", upstream: "http://192.168.1.10:3000/", host: "grafana.xenlor.com"
2024/03/21 12:05:50 [error] 224#224: *2553 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.1, server: grafana.xenlor.com, request: "GET / HTTP/2.0", upstream: "http://192.168.1.10:3000/", host: "grafana.xenlor.com"
